src/main/fc/fc_tasks.c

```diff
@@ -50,10 +49,12 @@
 #include "fc/cli.h"
 #include "fc/fc_dispatch.h"
 
+#ifdef USE_FC
```

Comment:
I prefer not to have `#ifdef`s around `#includes` - the minimal build time saving is not worth the loss of clarity.